lib/matplotlib/backend_managers.py

```diff
@@ -14,6 +18,197 @@
 from matplotlib.rcsetup import validate_stringlist
 import matplotlib.backend_tools as tools
 
+from matplotlib import is_interactive
+from matplotlib import rcParams
+from matplotlib.figure import Figure
+from matplotlib.backends import get_backend, backend as backend_name
```

Comment:
``get_backend``, no, as a function, but I think yes to ``backend as backend_name``, but no different from existing behaviour (I think) from when I wrote it.  After we had more backends converted to test with, we (me and @fariza) planned to test and improve the backend switching ability of matplotlib, and more than just switching, but having multiple backends active at the same time.